plugins/include-markdown/index.test.js

```diff
@@ -25,6 +25,23 @@ describe('include-markdown', () => {
       })
   })
 
+  test('include custom mdx components', () => {
```

Comment:
Added a test here, but as mentioned, couldn't just use `remark` directly as it doesn't recognize `jsx` nodes. So, instead, I'm making assertions against the AST that comes out after this plugin is run. In our use cases, this AST would ultimately be stringified, or rather rendered, by MDX, not `remark`.